python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -987,17 +987,22 @@ def test_tail(arctic_library):
     )
 
 
-@pytest.mark.parametrize("dedup", [True, False])
-def test_dedup(arctic_client, dedup):
+def test_dedup(arctic_client):
```

Comment:
Why this change?

Reply:
makes the test a bit faster on windows, due to not having to reinit the arctic_client on every option

Reply:
Creating `Arctic` shouldn't be slow at all. The fact that it's noticeable means we have a performance bug somewhere.

Please log an issue and put the link in a comment on this test.
Please also comment on the test that the manual parameterization is to workaround a performance problem and should not be emulated in other tests.

Reply:
created issue #1053 to address this